spark/src/main/kotlin/com/adevinta/spark/components/divider/DividerIntent.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 Adevinta
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.adevinta.spark.components.divider
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.adevinta.spark.SparkTheme
+
+/**
+ * DividerIntent is used to define the intent of the divider.
+ */
```

Comment:
Let's make a doc that avoid paraphrasing the name of the enum by adding what it'll affect and an advice on how to use it like here:
```suggestion
/**
 * Intent on the divider that will affect the background of the divider line.
 * Use [OutlineHigh] to get a higher contrast if needed.
 */
```